### ISSUE TITLE
[FEATURE REQUEST] Monthly calender view

### ISSUE THREAD

**Issue by wowawebdesign** [jweiland-net/events2#137]
It would be nice to have a big monthly calender view (like a Google calendar).

**Comment by froemken** [comment]
Yep. That is planned. We have a project customer who want's that, too.

**Comment by twurmehl** [comment]
any news on that?

**Comment by derBoogie** [comment]
Hi,
how far is the development of this feature?
I have a customer website that uses Events2 and the customer would like to have a month/week view in addition to the list view.
Regards TJ
